File: aifupan_project/ReviewAnalysis/Ai/Model/AudioaAlysesStatisticsDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.Ai.Model
{
    public class AudioaAlysesStatisticsDto
    {

        /// <summary>
        /// 视频开始时间 从0开始
        /// </summary>
        public long startTempTime {  get; set; }

        /// <summary>
        /// 视频结束时间 从0开始
        /// </summary>
        public long endTempTime { get; set; }

        /// <summary>
        /// 视频结束时间 从0开始
        /// </summary>
        public long endTempTimeTwo { get; set; }

        /// <summary>
        /// 视频开始时间 自然时间戳
        /// </summary>
        public long startVideoTime { get; set; }

        /// <summary>
        /// 视频结束时间 自然时间戳
        /// </summary>
        public long endVideoTime { get; set; }

        /// <summary>
        /// 视频的自然开始时间
        /// </summary>
        public DateTime? startTime { get; set; }

        /// <summary>
        /// 视频的自然结束时间
        /// </summary>
        public DateTime? endTime { get; set; }

        /// <summary>
        /// 视频的开始时间 HH:mm:ss
        /// </summary>
        public string textStart { get; set; }

        /// <summary>
        /// 视频的结束时间 HH:mm:ss
        /// </summary>
        public string textEnd { get; set; }

        /// <summary>
        /// 一段的在线人数
        /// </summary>
        public string renShu { get; set; }

        /// <summary>
        /// 一段的内容
        /// </summary>
        public string content {  get; set; }

    }
}

File: aifupan_project/fupan-server/replay-api/src/main/java/com/jiuyu/replay/api/logic/ai/dto/AudioaAlysesStatisticsDto.java
package com.jiuyu.replay.api.logic.ai.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;

/**
 * 音频分析统计数据传输对象
 *
 * @author liaoxin
 * @date 2025-01-27
 */
@Data
@Schema(description = "音频分析统计数据传输对象")
public class AudioaAlysesStatisticsDto implements Serializable {

    private static final long serialVersionUID = 1L;

    /**
     * 视频开始时间 从0开始
     */
    @Schema(description = "视频开始时间 从0开始")
    private Long startTempTime;

    /**
     * 视频结束时间 从0开始
     */
    @Schema(description = "视频结束时间 从0开始")
    private Long endTempTime;

    /**
     * 视频开始时间 自然时间戳
     */
    @Schema(description = "视频开始时间 自然时间戳")
    private Long startVideoTime;

    /**
     * 视频结束时间 自然时间戳
     */
    @Schema(description = "视频结束时间 自然时间戳")
    private Long endVideoTime;

    /**
     * 视频的自然开始时间
     */
    @Schema(description = "视频的自然开始时间")
    private Date startTime;

    /**
     * 视频的自然结束时间
     */
    @Schema(description = "视频的自然结束时间")
    private Date endTime;

    /**
     * 视频的开始时间 HH:mm:ss
     */
    @Schema(description = "视频的开始时间 HH:mm:ss")
    private String textStart;

    /**
     * 视频的结束时间 HH:mm:ss
     */
    @Schema(description = "视频的结束时间 HH:mm:ss")
    private String textEnd;

    /**
     * 一段的在线人数
     */
    @Schema(description = "一段的在线人数")
    private String renShu;

    /**
     * 一段的内容
     */
    @Schema(description = "一段的内容")
    private String content;
}